swingset/src/main/java/com/nqadmin/swingset/SSDBComboBox.java

```diff
@@ -1331,21 +1341,25 @@ public void setSelectedItem(final Object _value) {
 			if (selectedItem!=null) {
 				super.setSelectedItem(_value);
 				getEditor().selectAll(); // after we find a match, do a select all on the editor so if the user starts typing again it won't be appended
-				logger.debug("{}: Selected Index AFTER setSelectedItem()={}", () -> getColumnForLog(), () -> getSelectedIndex());
+				logger.debug("{}: Selected Index AFTER super.setSelectedItem()={}", () -> getColumnForLog(), () -> getSelectedIndex());
 			} else {
-				if (nullItem != null) {
-					super.setSelectedItem(nullItem);
-				} else {
-					logger.debug("{}: No matching list item found so not updating. Current editor text is '{}'", () -> getColumnForLog(), () -> getEditor().getItem().toString());
-				}
+				// Note that nullItem is null when allowNull is false.
+				// The next statement either selects
+				// the first item in combo, or a null.
+				super.setSelectedItem(nullItem);
+				// if (nullItem != null) {
+				// 	super.setSelectedItem(nullItem);
+				// } else {
+				// 	logger.debug("{}: No matching list item found so not updating. Current editor text is '{}'", () -> getColumnForLog(), () -> getEditor().getItem().toString());
```

Comment:
Maybe we can/should still do a logger.trace on the editor text?

Reply:
Clicked "Start a review" just to see how that was different from a comment.

Reply:
Put back log with adjusted text

Reply:
Thanks.